SuperblocksHQ/ethereum-studio
# Update analytics modal to only show when the templates modal is not showing

### Description of the Change

Updated the `Get permissions` analytics modal to only display when the template modal is not displaying.  

### Benefits

This removes some of the modals confusion when using Studio for the first time. 

### Verification Process

1. Open an incognito window and load the app
2. Make sure the analytics dialog is not appearing at the same time as the templates modal
3. When closing the templates modal, the analytics dialog should appear


### Github Issues

This resolves #68
